### DISCUSSION TITLE
configuration to segregate search and admin api's

### DISCUSSION THREAD

**Discussion by rgplvr** [meilisearch/product#417] [Feedback & Feature Proposal]
As i understand the Meilisearch use the same port for search and admin operations and it cant be configured. Providing a configuration to segregate this will enable users to control the exposure of apis using standard security group level controls 

for eg:
search : http://0.0.0.0:7700
admin apis: http://10.0.0.1:7702

**Comment by gmourier** [comment]
Hi @rgplvr 👋

Can you share more details? Why do you need to control the exposure of apis? Did you think about using a proxy on your side?

Thanks a lot for your answer 😇

**Comment by rgplvr** [comment]
Hi @gmourier 
proxies are definitely an option, but if we segregate the admin and data apis in different ports without using any other service we can restrict the admin apis using simple network rules and thereby reduce both the network and resource costs.
Regards,
Raj
